lib/python/fidasim/preprocessing.py

```diff
@@ -174,7 +174,16 @@ def check_inputs(inputs, use_abs_path=True):
               'calc_birth': zero_int,
               'calc_fida_wght': zero_int,
               'calc_npa_wght': zero_int,
-              'calc_neutron': zero_int}
+              'calc_neutron': zero_int,
+              'cell_split': zero_int}
+
+    if 'cell_split' in inputs:
+        if inputs['cell_split'] == 1:
+            schema['n_cells'] = zero_int
```

Comment:
I don't think `n_cells` should be a user supplied input but something that is calculated internally. Something like `max_cell_splits` would be appropriate though.

Reply:
I second @lstagner's comment.